bip-0183.md

```diff
@@ -0,0 +1,570 @@
+
+For all UTXOs that get added to the Utreexo merkle forest, a TTL info exists for it and includes information necessary for efficiently caching and requesting proofs.
+The TTL value provides information to determine which leaves should be cached and the death position is used to calculate which positions in the merkle forest we need to prove a block.
+
+| Field          | Type   | Description                                                                                                                                                          |
+|----------------|--------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| TTL            | varint | The time-to-live value of a leaf in the Utreexo merkle forest. The value is determined by the amount of leaves that were added to the accumulator since its creation |
+| death position | varint | The position in the Utreexo merkle forest when the leaf was removed                                                                                                  |
+
+#### Utreexo TTL
+
+| Field        | Type                | Description                                                                                                                                                                                                                                                    |
+|--------------|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| block height | uint32              | The time-to-live value of a leaf in the Utreexo merkle forest. The value is determined by the amount of leaves that were added to the accumulator since its creation                                                                                           |
+| length       | varint              | The length of the TTLs                                                                                                                                                                                                                                         |
+| TTLs         | vector of TTL infos | The TTL Info for the UTXOs that are added to the Utreexo merkle forest in blockchain ordering. See [BIP-0182](bip-0182.md#excluded-utxos-from-the-accumulator) for the UTXOs that are not added to the Utreexo merkle forest |
+
+### New Messages
+
+#### MSG_UTREEXO_PROOF
+
+`MSG_UTREEXO_PROOF` is all the data required for a CSN or archive node using the Utreexo accumulators to validate a Bitcoin block.
+
+Its `cmdString` for P2PV1 is `uproof`.
+Its [BIP324 P2PV2](https://github.com/bitcoin/bips/blob/master/bip-0324.mediawiki#user-content-v2_Bitcoin_P2P_message_structure) message type is `29`.
```

Comment:
I think these values should be reserved in bip 324 before being added to other bips, to help avoid conflicts. (At worst, perhaps bip-324 could be updated in this PR)